drivers/espi/espi_it8xxx2.c

```diff
@@ -1208,8 +1332,8 @@ static struct espi_it8xxx2_data espi_it8xxx2_data_0;
 static const struct espi_it8xxx2_config espi_it8xxx2_config_0 = {
 	.base_espi_slave = DT_INST_REG_ADDR_BY_IDX(0, 0),
 	.base_espi_vw = DT_INST_REG_ADDR_BY_IDX(0, 1),
-	.base_espi_queue1 = DT_INST_REG_ADDR_BY_IDX(0, 2),
-	.base_espi_queue0 = DT_INST_REG_ADDR_BY_IDX(0, 3),
+	.base_espi_queue1 = DT_INST_REG_ADDR_BY_IDX(0, 3),
+	.base_espi_queue0 = DT_INST_REG_ADDR_BY_IDX(0, 2),
```

Comment:
Change this so the index numbers are in order (queue0 first, then queue1)

Reply:
Done.